content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4/index.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Upgrade to Prisma 4'
+metaTitle: 'Upgrade to Prisma 4'
+metaDescription: 'Guides on how to upgrade to Prisma 4'
+tocDepth: 2
+toc: true
+---
+
+<TopBlock>
+
+Prisma 4 introduces a number of **breaking changes** if you are upgrading from an earlier version (`3.x.x`). This guide explains how this upgrade might affect your application and gives instructions on how to handle any changes.
```

Comment:
```suggestion
Prisma 4 introduces a number of **breaking changes** when you upgrade from an earlier version (`3.x.x`). This guide explains how this upgrade might affect your application and gives instructions on how to handle any changes.
```

Reply:
I'm not sure that's the best way to put it, but I'm trying to simplify the tense and avoid the gerund.

Reply:
Done